docs/how-tos/create-a-new-event.rst

```diff
@@ -175,20 +175,24 @@ The :term:`Event Definition` should be implemented in the corresponding subdomai
     # Location openedx_events/learning/signals.py
     # .. event_type: org.openedx.learning.course.enrollment.created.v1
     # .. event_name: COURSE_ENROLLMENT_CREATED
-    # .. event_description: emitted when the user's enrollment process is completed.
+    # .. event_key_field: enrollment.course.course_key
+    # .. event_description: Emitted when the user enrolls in a course.
     # .. event_data: CourseEnrollmentData
+    # .. event_trigger_repository: openedx/edx-platform
     COURSE_ENROLLMENT_CREATED = OpenEdxPublicSignal(
         event_type="org.openedx.learning.course.enrollment.created.v1",
         data={
             "enrollment": CourseEnrollmentData,
         }
     )
 
-- The event definition should be documented using in-line documentation with at least ``event_type``, ``event_name``, ``event_description``, and ``event_data``. This will help consumers understand the event and react to it. See :doc:`../reference/in-line-code-annotations-for-an-event` for more information.
+- The event definition should be documented using in-line documentation with at lest ``event_type``, ``event_name``, ``event_key_field``, ``event_description``, ``event_data`` and ``event_trigger_repository``. This will help consumers understand the event and react to it. See :doc:`../reference/in-line-code-annotations-for-an-event` for more information.
```

Comment:
```suggestion
- The event definition should be documented using in-line documentation with at least ``event_type``, ``event_name``, ``event_key_field``, ``event_description``, ``event_data`` and ``event_trigger_repository``. This will help consumers understand the event and react to it. See :doc:`../reference/in-line-code-annotations-for-an-event` for more information.
```